doc/ref/configuration/minion.rst

```diff
@@ -752,6 +752,30 @@ Statically assigns grains to the minion.
       cabinet: 13
       cab_u: 14-15
 
+.. conf_minion:: grains_blacklist
+
+``grains_blacklist``
+----------
```

Comment:
The dashed line here needs to be the same length as the previous line to avoid a Sphinx warning.